PubHub/PubHub.Common/Models/Books/AuthorInfoModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PubHub.Common.Models.Books
```

Comment:
Why did we put all the models in 'Books'?

Reply:
Because they're specifically used by books. No other entity uses them, as far as I know 🤔

Reply:
But can't an Operator change Author or Publisher information without ever knowing about how many books they have? 🤔

Reply:
Publisher, yes, author I would say no 🤔 
Authors will not have an endpoint either, I think.

The way we use Authors and Genres are mainly related to books, and as we discussed in a different discussion; Authors and Genres was to be selected via a dropdown multi-select.

I imagine that we can _tick_ them on or off depending on wheter they should be attached or not.
Or have I misunderstood the intend? (:

Reply:
Yes, I agree. I see that you moved the models around - I think it is good like that 🙂